lib/generators/mobile_workflow/install/templates/ability.rb

```diff
@@ -1,7 +1,9 @@
+# frozen_string_literal: true
```

Comment:
Adding `frozen_string_literal` manually because the generation of this file is triggered from the AppBuilder and it seems the fix implemented to add the line after creation does not work in this case. I need to investigate more, but for now I just add it here.

Reply:
Setting as resolved because I think it is cleaner to trigger the `abilities.rb` generation from the MobileWorkflow::Generators::InstallGenerator rather than from the AppBuilder, so I moved the file there and the frozen_string_literal is automatically added.